specification/cognitiveservices/data-plane/FormRecognizerReceipt/readme.md

```diff
@@ -0,0 +1,72 @@
+# Cognitive Services Form Recognizer Receipt SDK
```

Comment:
Don't we need to merge the entire change to data-plane/FormRecognizer instead of creating a new FormRecognizerReceipt?  From William: "Please follow what CV team has done for OCR. That is, two swagger files under the same data plane folder."

Reply:
done.